README.md

```diff
@@ -56,7 +58,8 @@ To inspect an existing language file (find untranslated strings), use this comma
 ```bash
 php artisan translatable:inspect-translations fr
 ```
-Only one language at a time, please.
+
+The `translatable:inspect-translations` commands only supports inspecting one language at a time.
```

Comment:
Please change to "The command only supports inspecting one language at a time." because we're in the context already and we don't need the plural form here.